guide/src/StyleGuide.js

```diff
@@ -141,7 +141,7 @@ export default React.createClass({
             main: {
                 background: "whitesmoke",
                 width: "100%",
-                margin: "50px 0px 50px 250px", 
+                margin: "50px 0px 0px 250px", 
```

Comment:
Ok, sorry, I see now, looking at this commit, that a lot more had changed than I thought:
https://github.com/cyverse/troposphere-ui/commit/d687535e5c7f0d89ba2c7e8ac71cf61a05ab03d1

You are right that we will need the left margin to clear the sidebar. 

The problem with using the content's margin to make room for the space bar is that we have one component with a style to account for a sibling it doesn't need to know about. This was a common layout but it isn't ideal. Say we want to make the sidebar a drawer if we want to push the content over with the sidebar we need to animate both the content margin and the sidebar together. With flexbox the main content doesn't have to care about it. If the drawer is open or closed it will just move over so all of the animation can live in the component we want to animate and the content can be blissfuly ignorant. 

The layout before that commit was in preparation of a drawer style sidebar. The auto margins were to center the max-width content within the space available.